website/docs/en/config/experiments.mdx

```diff
@@ -225,6 +225,53 @@ module.exports = {
 };
 ```
 
+## experiments.incremental
+
+<ApiMeta addedVersion="1.1.0-beta.0" />
+
+- **Type:** `undefined | boolean | Incremental`
+- **Default:** `undefined`
+
+```ts
+type Incremental = {
+  make?: boolean;
+  emitAssets?: boolean;
+  inferAsyncModules?: boolean;
+  providedExports?: boolean;
+  dependenciesDiagnostics?: boolean;
+  modulesHashes?: boolean;
+  modulesCodegen?: boolean;
+  modulesRuntimeRequirements?: boolean;
+  buildChunkGraph?: boolean;
+};
+```
+
+Whether to enable incremental rebuild to speed up the rebuild speed.
+
+`true` means enable incremental for all stages. `false` means disable incremental for all stages. Incremental can also be enabled only for specified partial stages:
+
+```ts
+module.exports = {
+  experiments: {
+    // enable incremental for all stages
+    incremental: true,
```

Comment:
Should we consider the NODE_ENV when setting this option?

```js
incremental: process.env.NODE_ENV === 'development',
```

I remember once trying to set it to `true` and it broke the production build.

Reply:
This should only effect on rebuild, so if it broke in production build it might be a bug

Reply:
get, I will try to provide a reproduction later